x-pack/test/functional/config.base.js

```diff
@@ -241,6 +241,20 @@ export default async function ({ readConfigFile }) {
           },
           kibana: [],
         },
+        test_filebeat_reader: {
```

Comment:
Is there anywhere else to put this besides the base config? Or does it need to be here because all the tests will need it?